.github/workflows/on_pr.yml

```diff
@@ -10,8 +10,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
-        os: [ubuntu-22.04, macos-13, macos-14]  # https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
+        os: [ubuntu-lateset, macos-13, macos-14, macos-latest]  # https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners
```

Comment:
@aprsa - does this match what you were suggesting?

Reply:
Yes, but without the typo. ;)